bigtable/docs/snippets.py

```diff
@@ -147,20 +149,19 @@ def test_bigtable_create_additional_cluster():
     cluster.delete()
 
 
-def test_bigtable_create_app_profile():
+def test_bigtable_create_reload_delete_app_profile():
+    import re
```

Comment:
Putting this here will avoid the sample containing the necessary imports. [START bigtable_app_profile_name]

Reply:
re is not needed for sample but is used for testing so left out of sample tags.